### PR TITLE
Czech Locale

### PR DESCRIPTION
**Pull request by geralch**
**What kind of change does this PR introduce?**
Czech Locale

**What is the current behaviour?**
Doesn't have Czech Locale

**What is the new behaviour?**
Now it has Czech Locale

**Checklist**:
- [ ] Documentation N/A
- [ ] Tests N/A
- [x] Ready to be merged

### CODE REVIEW

**Comment by arthurdenner** [comment]
@geralch, the failed check is due to your fork not being up-to-date with `master`.

It's not a blocker since it's only related to deployment of Storybook - all essentials checks are passing - but I thought you may be interested in fixing it and learn [how to sync a fork](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/syncing-a-fork).

File: src/locales/cs-CZ.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "todayButton": "Dnes",
+  "nextMonth": "Příští měsíc",
+  "previousMonth": "Předchozí měsíc",
+  "nextYear": "Příští rok",
+  "previousYear": "Minulý rok",
+  "weekdays": [
+    "Neděle",
```
Comment (knuhol) [inline src/locales/cs-CZ.json]: This should start with lower case

File: src/locales/cs-CZ.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    "pondělí",
+    "úterý",
+    "středa",
+    "Čtvrtek",
```
Comment (knuhol) [inline src/locales/cs-CZ.json]: Should start with lower case as well

File: src/locales/cs-CZ.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    "pátek",
+    "sobota"
+  ],
+  "months": [
+    "leden",
+    "Únor",
```
Comment (knuhol) [inline src/locales/cs-CZ.json]: Should start with lower case

File: src/locales/cs-CZ.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    "březen",
+    "duben",
+    "Smět",
```
Comment (knuhol) [inline src/locales/cs-CZ.json]: Incorrect, should be "květen"

File: src/locales/cs-CZ.json [hunk 5]
```diff
@@ -0,0 +1,30 @@
+{
+  "todayButton": "Dnes",
+  "nextMonth": "Příští měsíc",
+  "previousMonth": "Předchozí měsíc",
```
Comment (knuhol) [inline src/locales/cs-CZ.json]: Should be "Minulý měsíc"

**Comment by arthurdenner** [comment]
@geralch, I've asked Knut to review this PR since he's from Czech Republic and he noticed some mistranslations. Can you address them, please?

**Comment by geralch** [comment]
Thaks @arthurdenner, I'll change it as soon I can, sorry for the wait, I had a problem with my PC :)

**Comment by arthurdenner** [comment]
No problem, take your time!

**Comment by arthurdenner** [comment]
Hi @geralch! I hope you are well.

Do you need any help to update and finish this PR?

**Comment by arthurdenner** [comment]
Closed due to inactivity.

**Pull request closed without merging**